config_examples/jaeger.yaml

```diff
@@ -1,17 +1,17 @@
-exporters:
-  otlphttp:
-    endpoint: https://${env:HOST}/api/v2/otlp
-    headers:
-      Authorization: Api-Token ${env:API_TOKEN}
-
 receivers:
   jaeger:
     protocols:
-      thrift_http:
+      grpc:
```

Comment:
Why gRPC instead of thrift HTTP, is there not a way to test sending data over thrift?

Reply:
There is a way but that would mean that we'd have to basically write a thrift http exporter (implementing the `TraceDataSender` interface).

Also, we use `grpc` in our [docs](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/collector/use-cases/jaeger) - so it aligns it with what we have there.